.github/workflows/build-and-push-to-ghcr.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         if: github.ref != 'refs/heads/main'
         uses: docker/build-push-action@v5
         with:
-          context: ./server
+          context: server
```

Comment:
https://github.com/docker/build-push-action#git-context

Reply:
noticed that fixed it now builds